doc/extdev/index.rst

```diff
@@ -187,6 +187,84 @@ as metadata of the extension.  Metadata keys currently recognized are:
   output files can be used when the extension is loaded.  Since extensions
   usually don't negatively influence the process, this defaults to ``True``.
 
+.. _ext-i18n:
+
+Extension internationalization (`i18n`) and localization (`l10n`)
+-----------------------------------------------------------------
+
+.. versionadded:: 1.8
+
+An extension developped outside `Sphinx` own sources could obviously
+come with its own messages translations. This is allowed thanks :ref:`i18n-api`,
+and brieffly summarized in :func:`sphinx.locale.get_translation` help.
+
+In practice, you have to:
+
+#. Choose a name for your messages catalogues and template, which must be unique
+   (e.g. the name of your extension: ``'myextension'``).
+#. Mark in your extension `Python` sources all messages as translatable,
```

Comment:
Better probably not to use  single left ticks here, as [default_role](http://www.sphinx-doc.org/en/master/usage/configuration.html#confval-default_role) is not set-up in Sphinx doc/conf.py